demo/go.mod

```diff
@@ -7,11 +7,11 @@ toolchain go1.21.4
 require (
 	github.com/CosmWasm/wasmd v0.51.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.5 // indirect
-	github.com/cosmos/cosmos-sdk v0.50.6
+	github.com/cosmos/cosmos-sdk v0.50.7
 	github.com/cosmos/gogogateway v1.2.0 // indirect
 	github.com/cosmos/gogoproto v1.4.12
 	github.com/cosmos/iavl v1.1.2 // indirect
-	github.com/cosmos/ibc-go/v8 v8.0.0
+	github.com/cosmos/ibc-go/v8 v8.3.2
```

Comment:
Perhaps this broke the ica stuff? 🤷🏼

Reply:
I tried v8.0.0 and it does not make any difference, weird

Reply:
Likely v8.3.x is being pulled through the dependency tree.

Reply:
That, or the change comes from somewhere else (`cosmos-sdk`).